### PR TITLE
Bump cypress peer dependency to accept 3.x, 4.x, and 5.x

### PR DESCRIPTION
**Pull request by Sauloxd**
Hi there!

We are having a little problem when bumping to cypress 5, related to multiple cypress versions being installed together, and I think that's due to the multiple versions of cypress being defined in our yarn.lock.

Related issue:
- https://github.com/cypress-io/cypress/issues/6953
- https://github.com/cypress-io/cypress/issues/4595 and the [working fix for us](https://github.com/cypress-io/cypress/issues/4595#issuecomment-655741446)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +44,6 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "cypress": "^3.1.0"
+    "cypress": "^2.1.0 || ^3.1.0 || ^4.0.0 || ^5.0.0"
```
Comment (bierik) [inline package.json:47]: What about using `"cypress": "*"` ?

**Review by bierik — CHANGES_REQUESTED** [review]

**Comment by bierik** [comment]
@Sauloxd Could you please rebase against master. I updated all devDependencies and let the tests run against cypress 6.x. So https://github.com/4teamwork/cypress-drag-drop/pull/42/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R32 is obsolete them.

File: package.json (hunk 1 above)
Comment (Sauloxd) [reply package.json:47]: Could be!
The only caveat I see on using `*` is that what if `cypress 6` comes along with a major breaking change? I know it's not a real concern today, but I think it's nice for a library to keep the dependencies list with only known working deps. That way, when cypress 6 comes out (or something like that), you or the core contributors can manually approve that this package is still compatible with the new version, and publish a new version...
But I can change if you want me to, no problem!

Comment (bierik) [reply package.json]: 👍 Im fine with that.

**Pull request merged**